packages/gatsby-theme-docs/src/templates/page-content.js

```diff
@@ -16,42 +16,39 @@ const ContentCards = (props) => (
 );
 
 const PageContentTemplate = (props) => (
-  console.log(props),
```

Comment:
only this line was changed (remove that leftover console.log) but github shows the whole block as changed.